packages/react/src/primitives/PasswordField/ShowPasswordButton.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { ShowPasswordButtonProps } from '../types/';
+import { Button } from '../Button';
+import { ComponentClassNames } from '../shared/constants';
+import { IconVisibility } from '../Icon';
+import { Text } from '../Text';
+import classNames from 'classnames';
+
+export const ShowPasswordButton: React.FC<ShowPasswordButtonProps> = ({
+  onClick = () => {},
+}) => {
+  return (
+    <Button
+      className={classNames(
+        ComponentClassNames.FieldShowPassword,
+        ComponentClassNames.FieldGroupField
+      )}
+      onClick={onClick}
+    >
+      <Text className="sr-only">Show password</Text>
+      <IconVisibility size="large" alt="" />
```

Comment:
Should we swap icons when it is visible or invisible? `<IconVisibilityOff>`

Reply:
Yes, I forgot to do this. Will update.